tools/filters/grep.xml

```diff
@@ -1,9 +1,9 @@
-<tool id="Grep1" name="Select" version="1.0.1">
+<tool id="Grep1" name="Select" version="1.0.2">
```

Comment:
We also need to put this into https://github.com/mvdbeek/galaxy/blob/926268ce6e1acd3db4e51ceb72e689899c545b9c/lib/galaxy/tools/__init__.py#L195